specification/compute/Compute.Management/models.tsp

```diff
@@ -3316,6 +3453,23 @@ model AutomaticZoneRebalancingPolicy {
   rebalanceBehavior?: RebalanceBehavior;
 }
 
+/**
+ * The configuration parameters used to limit the number of virtual machines per availability zone in the virtual machine scale set.
+ */
+@added(Versions.v2025_04_01)
+model MaxInstancePercentPerZonePolicy {
+  /**
+   * Specifies whether maxInstancePercentPerZonePolicy should be enabled on the virtual machine scale set.
+   */
+  enabled?: boolean;
```

Comment:
Enums over booleans are preferred.

Reply:
We can look into it for the future but for now if possible We would like to keep it boolean - Compute already has a vast precedent of using booleans.

Reply:
It would be a good opportunity to fix issue on new APIs than to piggy back on existing legacy implementations.